Resources/templates/aliases.php.twig

```diff
@@ -1,13 +1,20 @@
 <?php
 
+{% for alias in aliases|slice(0,1) %}
+/**
+ * @file
+ * Drush site-alias configuration for {{ aliases["ac-site"] }}.
```

Comment:
**Question**  
Will this key `ac-site` always be available, or should we check for either `ac-site` or `site` depending on whether its Acquia Site Factory or Acquia Cloud?